### PR TITLE
Login button connected to spotify login screen

### PR DESCRIPTION
**Pull request by cavdarmerve**

### CODE REVIEW

File: BUMO/week-3-2/src/chat/menu/controller.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 module.exports = class MenuController {
-    constructor(ChatService) {
+    constructor(ChatService, $scope) {
```
Comment (dashersw) [inline BUMO/week-3-2/src/chat/menu/controller.js:2]: we don't use $scope here. I've never shown this, so please don't use code from somewhere else.

File: BUMO/week-3-2/src/chat/menu/template.pug [hunk 2]
```diff
@@ -2,7 +2,7 @@ h1 BUMO
 
 //button(ng-click="popoverVisible=!popoverVisible" outer-click="popoverVisible=false" outer-click-exclude="popover") Threads
     span(ng-if="$ctrl.count") &nbsp;({{ $ctrl.count }})
-button(ng-click="popoverVisible=!popoverVisible" outer-click="popoverVisible=false" outer-click-exclude="popover") Shuffle!
+button(ng-click="doSomething()" outer-click="popoverVisible=false" outer-click-exclude="popover") Shuffle!
```
Comment (dashersw) [inline BUMO/week-3-2/src/chat/menu/template.pug:5]: this should be `$ctrl.doSomething()`

File: BUMO/week-3-2/src/chat/side-panel/style.css [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+    background: lightgrey;
+    bottom: 270;
+}
+
+button {
+    display: block;
+    
+    
+    
+
```
Comment (dashersw) [inline BUMO/week-3-2/src/chat/side-panel/style.css:14]: we don't need this many empty lines here.

File: BUMO/week-3-2/src/chat/side-panel/style.css [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+}
+
+button {
+    display: block;
+    
+    
+    
+    
+    
+    }
```
Comment (dashersw) [inline BUMO/week-3-2/src/chat/side-panel/style.css:16]: you need an empty line on end of each file.

File: BUMO/week-3-2/src/chat/side-panel/template.pug [hunk 5]
```diff
@@ -1,11 +1,13 @@
 h4 Login with Spotify
-button(ng-click="popoverVisible=!popoverVisible" outer-click="popoverVisible=false" outer-click-exclude="popover") Login
+button(ng-click="doSomething()" outer-click="popoverVisible=false" outer-click-exclude="popover") Login
```
Comment (dashersw) [inline BUMO/week-3-2/src/chat/side-panel/template.pug:2]: the same here. `$ctrl.doSomething()`.

File: BUMO/week-3-2/src/chat/side-panel/style.css (hunk 4 above)
Comment (cavdarmerve) [reply BUMO/week-3-2/src/chat/side-panel/style.css:16]: removed.

File: BUMO/week-3-2/src/chat/side-panel/style.css (hunk 3 above)
Comment (cavdarmerve) [reply BUMO/week-3-2/src/chat/side-panel/style.css:14]: removed.

File: BUMO/week-3-2/src/chat/menu/template.pug (hunk 2 above)
Comment (cavdarmerve) [reply BUMO/week-3-2/src/chat/menu/template.pug:5]: changed.

File: BUMO/week-3-2/src/chat/menu/controller.js (hunk 1 above)
Comment (cavdarmerve) [reply BUMO/week-3-2/src/chat/menu/controller.js:2]: removed

**Pull request merged**